types/openid-client/index.d.ts

```diff
@@ -69,6 +70,14 @@ export class Client {
         readonly max_age?: number;
     }): Promise<TokenSet>;
 
+    oauthCallback(redirectUri: string, parameters: {}, checks?: {
+        readonly response_type?: string;
+        readonly state?: string;
+        readonly nonce?: string;
+        readonly code_verifier?: string;
+        readonly max_age?: number;
+    }): Promise<TokenSet>;
```

Comment:
Here we should drop max_age and nonce. 

See 
> If you're using pure OAuth 2.0 then client.oauthCallback(redirectUri, parameters[, checks[, extras]]) is the OAuth 2.0 variant of this method, it has the same signature with the exception of checks only supporting code_verifier, state and response_type.

https://github.com/panva/node-openid-client/blob/master/docs/README.md#clientcallbackredirecturi-parameters-checks-extras

... and see also the code: https://github.com/panva/node-openid-client/blob/8d15d302f7fa53d953f032d07fc03b7f0a008285/lib/client.js#L488

Reply:
Thank you for the catch!